#161342795 setup react router version 4

ucheg6 opened the pull request:
#### What does this PR do?
setup react router version  4
#### Description of Task to be completed?
- install and configure react router
- create a component to test the route configuration

#### How should this be manually tested?
- Git clone the repository
- cd into the `haven-ah-frontend` folder
- run `npm run start:dev`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#161342795](https://www.pivotaltracker.com/story/show/161342795)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

victor-nweze-deel committed:
ch(react setup): Setup an empty react project and host it on heroku

setup and configure webpack
configure babel to transpile to es5
configure eslint with airbnb styleguide

PriscillaSam commented:
This looks good, guys @ProfJigsaw @ucheg6

seyi-adeleke commented on src/components/HomePage.js:
```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+class HomePage extends Component {
```
Make this a pure component

ucheg6 committed:
ch(react-routers): Setup react router version 4 (React-router-dom)

install and configure react router creating a dummy route for homepage
create a component to test the route configuration
